### ISSUE TITLE
Not working with Raspbian Stretch

### ISSUE THREAD

**Issue by duchenpaul** [Motion-Project/motion#545]
I installed the motion and encountered Segment fault while trying to launch it

1. version: v6.4.08
2. installed as a package or compiled from sources [deb, rpm, git, other]: apt-get
3. motion standalone
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: mjpeg
5. hardware [x86, ARM, other]: rpi official camera, pi zero w
6. operating system [Linux (which), FreeBSD, other]: Stretch

**Comment by jmtatsch** [comment]
Do you use a fresh config file or one from an old version?
do you use /dev/video0 or mmal as video source?
What does the log say?

**Comment by duchenpaul** [comment]
Hi @jmtatsch 
- Do you use a fresh config file or one from an old version?
_A fresh config file, I even tried to re-flash the system for several times, it just fails still, but it is all good when I install on jessie, same device._

- do you use /dev/video0 or mmal as video source?
_I think it should be mmal, the rapsberry pi official cam._

- What does the log say?
_Here is what I get when I finished installing, as it fails directly, so there is no log, default log should be `/tmp/motion.log`, however no such file_
```
[ 11:05 - 192.168.31.228  ]
pi@Venus ~ $ sudo apt-get install motion
Reading package lists... Done
Building dependency tree       
...
Done.
Processing triggers for systemd (232-25+deb9u1) ...
Processing triggers for man-db (2.7.6.1-2) ...

[ 11:19 - 192.168.31.228  ]
pi@Venus ~ $ motion
Segmentation fault
[ 11:19 - 192.168.31.228  ]
pi@Venus ~ $ 
[ 11:35 - 192.168.31.228  ]
pi@Venus /etc/motion $ sudo dpkg -l | grep motion
ii  motion                          3.2.12+git20140228-4+b2           armhf        V4L capture program supporting motion detection

```

```
My running environment:
Device: raspberry pi zero w
Camera: Pi NoIR Camera V2
OS: 
Linux Venus 4.9.41+ #1023 Tue Aug 8 15:47:12 BST 2017 armv6l GNU/Linux

Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 9.1 (stretch)
Release:	9.1
Codename:	stretch

```

**Comment by genius3000** [comment]
3.2.12 is quite an old version now and doesn't support the Pi MMAL cam (unless they are using the unofficial fork that initially added Pi cam support).  
I would recommend uninstalling that and install the 4.1 .deb from Releases (specifically the pi_stretch... one): [Release 4.1](https://github.com/Motion-Project/motion/releases/tag/release-4.1)

**Comment by Mr-Dave** [comment]
The version of Motion included with Stretch via apt is 4.0.1 so it would appear that the apt configuration on the machine is not correct which could cause numerous problems.  

As a reminder there was the following note when you opened this issue at the top of the screen.  
"Before you open an issue please review the [contributing guidelines](https://github.com/Motion-Project/motion/blob/master/CONTRIBUTING.md) for this repository. "
